log-collector/pkg/fluentd/fluentd.go

```diff
@@ -9,6 +9,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Any reason why we are using `k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/api/errors` instead of`k8s.io/apimachinery/pkg/api/errors`